taskd_services/http/api.py

```diff
@@ -77,22 +81,35 @@ def get_data_path(self, *path) -> str:
     def delete(self):
         env = os.environ.copy()
         env["TASKDDATA"] = TASKD_DATA
-        command = [TASKD_BINARY, "remove", "user", self.org_name, self.user_name]
+        command = [TASKD_BINARY, "remove",
+                   "user", self.org_name, self.user_key]
         delete_proc = subprocess.Popen(
             command, stdout=subprocess.PIPE, stderr=subprocess.PIPE, env=env,
         )
         result = delete_proc.wait()
+
+        del_proc_output = delete_proc.communicate()[
+            0].decode("utf-8").split("\n")
+        result = delete_proc.wait()
```

Comment:
This looks like it's a duplicate of line 89 above -- you probably only need to wait once, I think.

Reply:
Yes! My bad